internal/configs/test_file.go

```diff
@@ -1010,17 +1010,22 @@ func checkForDuplicatedOverrideModules(modules []*OverrideModule) hcl.Diagnostic
 	return diags
 }
 
+// Schema defining the structure for test configurations in tofu tests.
```

Comment:
```suggestion
// testFileSchema defines the structure of test file configuration for tofu tests.
```

Could you please tweak other global var comments to follow the same style as above?

Reply:
@ollevche Done ✅